WalletWasabi.Fluent/Views/Settings/BitcoinTabSettingsView.axaml

```diff
@@ -9,12 +9,6 @@
              x:CompileBindings="True">
   <StackPanel Classes="settingsLayout">
 
-    <DockPanel>
-      <TextBlock Text="Network" />
```

Comment:
why remove this setting?

Reply:
Because you can change it anymore and it is not persisted anywhere anymore.